docs/pages/foundations/messaging/a_messaging_story.js

```diff
@@ -0,0 +1,202 @@
+// @flow strict
+import { Heading, Text, Flex, Box, Image } from 'gestalt';
+import React, { type Node } from 'react';
+import Page from '../../../docs-components/Page.js';
+import PageHeader from '../../../docs-components/PageHeader.js';
+
+type StoryItemProps = {|
+  heading: string,
+  imageUrl: string,
+  imageWidth: number,
+  imageHeight: number,
+  text: string | Node,
+  format?: 'twoColImgLeft' | 'twoColImgRight' | 'oneCol',
+|};
+
+function StoryItem({
+  heading,
+  imageUrl,
+  imageWidth,
+  imageHeight,
+  text,
+  format = 'oneCol',
+}: StoryItemProps): Node {
+  const img = (
+    <Box
+      width="100%"
+      padding={4}
+      display="flex"
+      flex="grow"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <Box maxWidth={format === 'oneCol' ? '100%' : 320} width="100%">
+        <Image alt="collage" src={imageUrl} naturalWidth={imageWidth} naturalHeight={imageHeight} />
+      </Box>
+    </Box>
+  );
+  const contentMobile = (
+    <Box width="100%" padding={4} lgDisplay="none" display="block">
+      <Flex direction="column" gap={2}>
+        <Heading size="400">{heading}</Heading>
+        <Text>{text}</Text>
+      </Flex>
+    </Box>
+  );
+  const contentDesktop = (
+    <Box width="100%" padding={4} lgDisplay="block" display="none">
+      <Flex direction="column" gap={2}>
+        <Heading size="400">{heading}</Heading>
+        <Text>{text}</Text>
+      </Flex>
+    </Box>
+  );
+  const lockup =
+    format === 'twoColImgRight' ? (
+      <React.Fragment>
+        {contentDesktop}
+        {img}
+        {contentMobile}
+      </React.Fragment>
+    ) : (
+      <React.Fragment>
+        {img}
+        {contentDesktop}
+        {contentMobile}
+      </React.Fragment>
+    );
+  return (
+    <Box
+      display="flex"
+      direction="column"
+      lgDirection={format === 'oneCol' ? 'column' : 'row'}
+      alignItems="center"
+    >
+      {lockup}
+    </Box>
+  );
+}
+
+export default function MessagingStoryPage(): Node {
+  return (
+    <Page title="A messaging story">
+      <PageHeader
+        name="A messaging story"
```

Comment:
Totally in love with this page, amazing work @hectoid!!

Reply:
❤️